src/Cache/KvCacheAdapter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baldinof\RoadRunnerBundle\Cache;
+
+use Spiral\Goridge\RPC\RPC;
+use Spiral\RoadRunner\KeyValue\Factory;
+use Symfony\Component\Cache\Adapter\Psr16Adapter;
+
+class KvCacheAdapter extends Psr16Adapter
```

Comment:
```suggestion
/**
 * @internal
 */
final class KvCacheAdapter extends Psr16Adapter
```

Unless there is actual use case to allow extension and creation from outside of the bundle.

Reply:
Applied.